src/NzbDrone.Core/Indexers/Definitions/AnimeWorld.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
+using NLog;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Indexers.Definitions.UNIT3D;
+using NzbDrone.Core.Messaging.Events;
+
+namespace NzbDrone.Core.Indexers.Definitions
+{
+    public class AnimeWorld : Unit3dBase
+    {
+        public override string Name => "AnimeWorld";
+        public override string BaseUrl => "https://animeworld.cx/";
+
```

Comment:
Can you add the Indexer Description override here as well? Thanks

Reply:
done ^^